src/modelinstance.cpp

```diff
@@ -305,7 +306,7 @@ static Status applyLayoutConfiguration(const ModelConfig& config, std::shared_pt
     }
 
     OV_LOGGER("ov::Model: {}, model->outputs()", reinterpret_cast<void*>(model.get()));
-    for (const ov::Output<ov::Node>& output : model->outputs()) {
+    for (ov::Output<ov::Node>& output : model->outputs()) {
```

Comment:
is this change required?

Reply:
yes, we are changing the outputs by adding name

Reply:
but not in this for loop, i dont think we need to remove const in this for loop